tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -33,7 +33,19 @@
 """Schema separately parses translation to account for multiple valid formats
 github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md
 #defining-translations"""
-_TRANSLATION_SCHEMA = syaml.Str() | syaml.Any()
+_TRANSLATION_SCHEMA = syaml.Str() | syaml.MapPattern(
+    syaml.Str(),
```

Comment:
Can we be stricter about the keys? We have rules for case sensitivity and format for fields.

Reply:
done

Reply:
this key can also use the field regex, no?